File: URLShortener.Front/src/app/models/SettingModel.ts
export class SettingModel{
    constructor(id: number, account_id: string, key: string, title: string, description: string, isActive: boolean){
        this.id = id;
        this.account_id = account_id;
        this.key = key;
        this.title = title;
        this.description = description;
        this.isActive = isActive;
    }
    public id: number;
    public account_id: string;
    public key: string;
    public title: string;
    public description: string;
    public isActive: boolean;
}

File: URLShortener.Back/URLShortener.Models/SettingModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace URLShortener.Models
{
    public class SettingModel
    {
        public int Id { get; set; } = 0;
        public string Account_Id { get; set; } = string.Empty;
        public string Key { get; set; } = string.Empty;

        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public bool IsActive { get; set; } = false;
    }
}
